share/functions/fish_vi_key_bindings.fish

```diff
@@ -69,7 +69,7 @@ function fish_vi_key_bindings --description 'vi-like key bindings for fish'
     bind -s --preset :q exit
     bind -s --preset -m insert \cc cancel-commandline
     bind -s --preset -M default h backward-char
-    bind -s --preset -M default l forward-char
+    bind -s --preset -M default l forward-char or accept-autosuggestion
```

Comment:
There are more things in the vi-bindings that use "forward-char".

It's probably okay for these to not accept the entire line, like you've done here.

Possibly "a" might? I'm not sure as I don't use vi-bindings (and we could really use people who do to work on them).

Reply:
Yeah, I've looked at the other bindings and most of them looked like where were meant to leave the cursor roughly where it was.